src/clients/noizd.ts

```diff
@@ -21,7 +21,7 @@ export type NOIZDClient = {
 }
 
 const mapAPIListingToTrackID = (listing: any): string => {
-  return ethereumTrackId(listing.nft.metadata.contract, listing.id);
+  return `noizd/${listing.id}`
 };
```

Comment:
hmm what was going on here? was it like ethereum/uuid and so kinda broken before already? will this result in changing track ids tho?

Reply:
I see that some noizd tracks are prefixed with noizd/ and others are prefixed with ethereum/. Not exactly sure if always going with noizd/ is a good solution but not all noizd nfts are on ethereum or on chain so I think it's better to use the hardcoded noizd/ prefix here?

Reply:
yeah basically all tracks start with the noizd/ prefix premint but then once the nft is minted, the id gets migrated to ethereum/ and changes

(this is actually problematic too, since ids change and tracks will get lost from favs/playlists, but i wouldn't change it from how it currently is since there may be side effects in other processors)

Reply:
That's a pretty surprising change in ids. 
Would it remove the need for that transformation from `noizd`->`ethereum` if we just added the platform as part of the track id, along with the chain and listing id?

i.e.
```
`ethereum/noizd/${listing.id}`
```

Or does noizd publish to different chains, making the `ethereum` part inaccurate?

Reply:
But maybe the ideal form of an immutable ID here is to just have the `chain/address`, especially seeing as platform information isn't strictly part of all contracts.

Like you said though @musnit, maybe we deal with this problem later but it'll be useful to know what we're aiming for with IDs in general (due to similar considerations in the Artist and ArtistProfile part of the codebase).